### PR TITLE
Feature/upstream binance/upstream model

### PR DESCRIPTION
**Pull request by dyakubovich**

### CODE REVIEW

**Comment by seletskiy** [comment]
@ritesh-skyach: Hey, can you check that it works with your UI changes?

File: db/schema.rb [hunk 1]
```diff
@@ -98,7 +98,20 @@
   add_index "deposits", ["tid"], name: "index_deposits_on_tid", using: :btree
   add_index "deposits", ["type"], name: "index_deposits_on_type", using: :btree
 
+  create_table "identities", force: :cascade do |t|
```
Comment (ritesh-skyach) [inline db/schema.rb]: @dyakubovich Where is migration for this table ?

File: db/schema.rb [hunk 2]
```diff
@@ -172,6 +185,18 @@
 
   add_index "payment_addresses", ["currency_id", "address"], name: "index_payment_addresses_on_currency_id_and_address", unique: true, using: :btree
 
+  create_table "tokens", force: :cascade do |t|
```
Comment (ritesh-skyach) [inline db/schema.rb]: @dyakubovich Where is migration for this table ?

**Comment by ritesh-skyach** [comment]
@seletskiy Yes, I will work on these changes

File: db/migrate/20180905093248_create_upstreams.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class CreateUpstreams < ActiveRecord::Migration
+  def change
+    create_table :upstreams do |t|
+      t.string :provider, limit: 32, null: false, index: {unique: true}
+      t.integer :enabled, limit: 1, null: false, default: 0
```
Comment (ritesh-skyach) [inline db/migrate/20180905093248_create_upstreams.rb]: Please change the `enabled` field type to `boolean`

**Pull request merged**